### PR TITLE
Location search

### PR DESCRIPTION
**Pull request by temilolu97**
#### How should this manually be Tested?
- `git clone` __https://github.com/devcareer/Dae-Alright-Backend__
- `cd` **Dae-Alright-Backend**
- `git checkout <branch-name>`
- `npm install`
- `npm test`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
   "scripts": {
     "build": "npm run clean && npm run build-babel",
     "build-babel": "babel -d ./build ./src -s",
-    "clean": "rm -rf .nyc_output build coverage && mkdir build",
+    "clean": "rimraf .nyc_output build coverage && mkdir build",
```
Comment (TundeMercy) [inline package.json:24]: Please undo this change

File: src/controllers/locationSearch.controller.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import sequelize from 'sequelize';
+import database from '../database/models';
+import { successResponse, errorResponse, serverError } from '../helpers/response';
+
+const getByLocation = async (req, res) => {
+    try { 
+        const vendors = await database.Vendor.findAll({
+            where: {
+                location: req.query.location,
```
Comment (TundeMercy) [inline src/controllers/locationSearch.controller.js:9]: This might not be optimal, the `location` value might not be strictly equal to `query.location`, can't we use something like `location like` or `location includes`. Research more on it please.

**Review by TundeMercy — COMMENTED** [review]
Please fix all hound and other issues pointed out and squash all commit into one.

File: src/controllers/locationSearch.controller.js (hunk 2 above)
Comment (iphyokafor) [reply src/controllers/locationSearch.controller.js:9]: Whatever location the user inputs which is the query location, if it matches any location stored in the database then the user gets the results if not a message pops up. However, other options relating to the query location could come up as alternative options that's where we can add the other attributes ( like or include) if possible.

**Review by iphyokafor — COMMENTED** [review]
Kindly fix the lint errors. Did you test it on postman after solving the previous problem?

Well done.

**Comment by supercede** [comment]
Hi, Please squash your commits into one, fix the hound errors,  correct/resolve the reviews pointed out by Tunde and push your changes.

**Review by Endowmissy — APPROVED** [review]

**Review by iphyokafor — APPROVED** [review]

**Comment by maestro-1** [comment]
Please fix all hound issues and squash commits to one
